### ISSUE TITLE
Tag 701 break focus component

### ISSUE THREAD

**Issue by celogeek** [jitsi/jicofo#676]
Hi,

I try the nightly patch on my setup. When I install this one, it breaks my installation.

https://github.com/jitsi/jicofo/pull/672

I have a "secure domain" installation, that mean I need to identify myself to create a room. My instance is not open to anyone.

I've got a FocusComponent with a secret that matches the JICOFO_SECRET.

What I try to do is to change it to:
```
Component "focus.MY_MEET_DOMAIN" "client_proxy"
  target_address = "focus@auth.MY_MEET_DOMAIN"
```

And then I run:
```
 prosodyctl mod_roster_command subscribe focus.MY_MEET_DOMAIN focus@auth.MY_MEET_DOMAIN
```

I try to see if anything else was required to move on, I haven't find anything else on debian script.

With this new setup, I don't understand why I still need to defined the JICOFO_SECRET.

Do I have an extra step in the case of "secure domain" I miss ?

My issue is when I try to log on, It refuses with an error message on the creation of the room, with bad domain in the logs.

The error code is 'connection.GET_SESSION_ID_ERROR', with in the logs:
```
{authenticationError: "not-authorized", message: "not authorized user domain"}
```

**Comment by celogeek** [comment]
I use the https://github.com/jitsi/jitsi-meet-debian-meta:

```
==> jicofo <==
701

==> jitsi-meet-web <==
4656

==> jitsi-videobridge <==
2.1-417-g6112079e
```

**Comment by bgrozev** [comment]
Hey @celogeek. This is fixed #674, no changes needed on your end.

**Comment by celogeek** [comment]
Ok nice. I will try it when you release it.
I hold on the nightly on jicofo 700 on arch linux for now.

the removal of the component is a breaking change.

**Comment by bgrozev** [comment]
> I hold on the nightly on jicofo 700 on arch linux for now.

Does this use a separate package not based on our debian packages?

**Comment by celogeek** [comment]
I use the jitsi-meet-debian-meta repo to group update together. I fetch the debian package then repack them for Arch Linux.
I can skip some version or apply temp fix if needed during the process.

I also have a build from source version, which I can hold on also to avoid releasing to Arch User a set of package that doesn't works. I always test it first, but I miss that bug (have to wait 1 or 2 min before it occur), so I have a not working version during 1 day.

**Comment by celogeek** [comment]
I've test the latest version. I check what changes in the debian installer and I spot an error:

The jicofo.conf look like this with the defaut installation (fresh nightly one on ubuntu):

```
jicofo {
}
xmpp: {
  client: {
    client-proxy: focus.meet.celogeek.com
  }
}
```

And when I try this on my arch installation, it failed to authenticate.
I fix it by filling the reference.conf file (which is copied as a jicofo.conf for default value with comments).
So it looks like:
```
jicofo {
  xmpp: {
    client: {
      client-proxy: focus.meet.celogeek.com
    }
  }
}
```

I also see that turn setup has changed (turncredential), and I have issues with it (laggy). Try to fix it also.

But now I have report the JICOFO PORT and SECRET, and I use the focus user with prosody !

**Comment by bgrozev** [comment]
> I fix it by filling the reference.conf file

Thanks for the report! Fixed in #685
